ci-operator/step-registry/assisted/baremetal/test/assisted-baremetal-test-ref.yaml

```
@@ -15,17 +15,26 @@ ref:
     default: "false"
     documentation: If the value is equal to "true", run only the upgrade tests
   - name: TEST_TYPE
-    default: "list"
+    default: fixed
     documentation: |-
       Specifies the type of test to run:
       * 'suite' - Run a full test suite. If chosen, 'TEST_SUITE' and 'TEST_PROVIDER' can be specified.
       * 'list' - Run a specific list of tests. 'TEST_LIST' specified.
+      * 'fixed' - Run a fixed test list.
       * 'none' - Don't run any tests.
```

```suggestion
      * 'custom' - Run a specific list of tests. 'TEST_LIST' must be specified.
      * 'minimal' - Run the minimal test list.
      * 'extensive' - Run the extensive test list.
      * 'none' - Don't run any tests.
```

and let's remove `FIXED_TEST_LIST` as it looks redondant